com.thincast.client.json

```
@@ -0,0 +1,292 @@
+{
+    "app-id": "com.thincast.client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
```

Update runtime to 5.14
```suggestion
    "runtime-version": "5.14",
```

---

Due to a bug in Qt 5.14 regarding NTLM authentication we have to stick with 5.13 for now.